rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/reporting/rules/CreateJavaIgnoredFilesReportRuleProvider.java

```diff
@@ -35,6 +35,12 @@
 {
     public static final String TITLE = "Ignored Files";
     public static final String TEMPLATE_REPORT = "/reports/templates/ignored_files.ftl";
+    public static final String DESCRIPTION = "This report lists the files which were found in the application,\n" +
+            "                    but based on certain rules and the Windup configuration, both built-in and local,\n" +
+            "                    they were not processed.\n" +
+            "                    See the <code>--userIgnorePath</code> Windup option in\n" +
+            "                    <a href=\"http://windup.github.io/windup/docs/latest/html/WindupUserGuide.html#command-line-arguments\"\n" +
+            "                       >Windup User Guide</a>.";
```

Comment:
we might need to add specific anchors for option argument in WIKI doc to point directly into it.